scripts/install-from-shrinkzip.sh

```diff
@@ -1,12 +1,13 @@
 #!/usr/bin/bash
 
-set -x
+set -ex
 
 # This script is necessary to produce the .zip files to submit to mozilla store
 # Otherwise the repo is too big
 
 out=../trex-from-zip
 
+rm -r $out
```

Comment:
I'm always afraid of running an `rm` without ensuring in which path you're running the script :) or without putting the `-i` flag.

If this script is meant for the package maintainer, you should assume they need to understand what each step does, thanks!

Reply:
I removed that part, cause I was scary too :D